nuxeo-distribution/nuxeo-war-tests/README.md

```diff
@@ -0,0 +1,8 @@
+This project holds the functional tests to be run on a
+Tomcat with Nuxeo deployed as a WAR inside
```

Comment:
Point to the related documentation ? https://doc.nuxeo.com/nxdoc/deploying-as-a-standard-static-war